src/client.ts

```diff
@@ -851,6 +905,56 @@ const versionIsNewer = (oldV: string | undefined, newV: string): boolean => {
     : newV.localeCompare(oldV, 'en') > 0
 }
 
+interface ClockRequest {
+  id?: string
+  name?: string
+  present: Date | string
+}
+
+interface ClockResponseJSON {
+  id: string
+  link: string
+  present: string
+  status: string
+}
+
+export interface ClockResponse {
```

Comment:
Why export? Do `withClock` and `advance` not hide all of this?

Reply:
It's returned by `syncClock`, which is a public method.  I could make `syncClock` private, I suppose, but it's generally kind of annoying to have a public method that returns a private data type, because then anyone using it has to resort to hacky stuff like `Awaited<ReturnType<typeof Tier['syncClock']>>` to reference the return type.

Reply:
Moved to private methods on 4b2b1d555c2376b3761a54c9d061f6407feed9bd